tabpy/models/deploy_models.py

```diff
@@ -48,3 +48,6 @@ def install_dependencies(packages):
     for filename in os.listdir(directory):
         subprocess.run([py, f'{directory}/{filename}', config_file_path]
```

Comment:
Did you smoke test this with a custom config? I'm not sure about specifying absolute vs relative path for the custom config anymore

Reply:
I updated deploy_and_evaluate integration test to use the custom config.

Reply:
I see the tests now modify the state.ini file, shouldn't they write to a different state file so that running the tests doesn't add the models endpoints?

Reply:
state.ini is not in the repo anymore and is created as a new file for each test from state.ini.template.